airflow/www/templates/analytics/google_analytics.html

```diff
@@ -0,0 +1,9 @@
+<script>
```

Comment:
Missing license headers on these new files. If you use https://github.com/apache/airflow/blob/master/CONTRIBUTING.md#pre-commit-hooks it will add them for us.

Reply:
Ah very nice, just added it.